job/runner.go

```diff
@@ -79,3 +93,14 @@ func (j *Job) Run() error {
 	cancel()
 	return err
 }
+
+// RunAnd Cleanup executes a command and clean up the job and pods.
```

Comment:
__[Golint]__ comment on exported method Job.RunAndCleanup should be of the form "RunAndCleanup ..." ([link](https://sider.review/gh/repos/169710725/pulls/25#issue-4259319))

<sub>posted by Sider</sub>